samples/azure/azure-service-fabric-routing/sample.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Azure Service Fabric Partition Aware Routing
+reviewed: 2017-02-14
+component: Core
+related:
+- nservicebus/azure-service-bus
+---
+
+
+## Prerequisites
+
+1. Azure Service Fabric [dev cluster](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-get-started) running configured with 5 nodes.
+2. An Azure Service Bus namespace that can be used for communication between the instances.
+3. A machine level environment variable called "AzureServiceBus.ConnectionString" that contains the manage connectionstring to the Azure Service Bus namespace.
```

Comment:
Should we add note that USER level user setting is not enough? I think it's easy to potentially miss the `machine level` part.